Documentation/quickstart/compiling.rst

```diff
@@ -0,0 +1,54 @@
+.. include:: /substitutions.rst
```

Comment:
This document feels a bit repetitive over the quickstart. Maybe we don't need it?

Reply:
@v01d I would like to keep it for now. Can we leave it in for this PR, and remove it in another PR if necessary? The reason it's here is to give people who are not doing the Quickstart a way to flow through detailed instructions.

Reply:
Sure, no problem.